website/database.py

```diff
@@ -7,6 +7,7 @@
 TOKENS = dynamo.Table(storage, 'tokens', 'id')
 PROGRAMS = dynamo.Table(storage, 'programs', 'id', indexed_fields=['username'])
 CLASSES = dynamo.Table(storage, 'classes', 'id', indexed_fields=['teacher', 'link'])
+PREFERENCES = dynamo.Table(storage, 'preferences', 'id')
```

Comment:
I am not sure I love the name preferences because we have been calling it customizations and preferences is quite generic. Shall we rename the table customization or class-customization?

Reply:
That would indeed be more straight forward. Will rename all variables (and the db) into customizations.